api/src/iot/tests/test_parser_ais_masten.py

```diff
@@ -0,0 +1,330 @@
+        assert self.actual == [self.expected]
+
+
+@pytest.mark.django_db
+class TestImportSensor:
+
+    @property
+    def actual(self):
+        return [
+            Device2Serializer(device).data
+            for device in models.Device2.objects.all()
+        ]
+
+    expected_1 = {
+        'active_until': '2050-01-01',
+        'contains_pi_data': True,
+        'datastream': '',
+        'legal_ground': 'In de rol van vaarwegbeheerder op basis van de binnenvaartwet.',
+        'location': {'latitude': 4.899393, 'longitude': 52.4001061},
+        'location_description': None,
+        'observation_goal': 'Vaarweg management',
+        'owner': {
+            'name': 'ais masten',
+            'email': 'programmavaren@amsterdam.nl',
+            'organisation': 'Gemeente Amsterdam',
+        },
+        'privacy_declaration': 'https://www.amsterdam.nl/privacy/specifieke/\
+privacyverklaringen-b/vaarwegbeheer/',
+        'regions': [],
+        'themes': ['Mobiliteit: auto'],
+        'type': 'Overig',  # needs to be checked why not Feature
+        'reference': '9',
+    }
+
+    expected_2 = {
+        'active_until': '2050-01-01',
+        'contains_pi_data': True,
+        'datastream': '',
+        'legal_ground': 'In de rol van vaarwegbeheerder op basis van de binnenvaartwet.',
+        'location': {'latitude': 4.899393, 'longitude': 52.4001061},
+        'location_description': None,
+        'observation_goal': 'Vaarweg management',
+        'owner': {
+            'name': 'ais masten',
+            'email': 'programmavaren@amsterdam.nl',
+            'organisation': 'Gemeente Amsterdam',
+        },
+        'privacy_declaration': 'https://www.amsterdam.nl/privacy/specifieke/\
+privacyverklaringen-b/vaarwegbeheer/',
+        'regions': [],
+        'themes': ['Mobiliteit: auto'],
+        'type': 'Overig',  # needs to be checked why not Feature
+        'reference': '10',
+    }
+
+    def test_import_sensor(self, sensor_data):
+        # Basic check for importing sensor data
+        owner = import_utils.import_person(sensor_data.owner)
+        import_utils.import_sensor(sensor_data, owner)
+        assert self.actual[0] == self.expected_2
+
+    def test_import_sensor_from_parse_ais_masten_success(self, am_data):
+        """
+        provide a dict from the ais_masten api and call
+        the parser of the ais_masten to get a sensor.
+        call the import_sensor and expected it to be imported.
+        """
+        parser = import_utils_apis.parse_ais_masten
+        sensor_list = list(parser(am_data))
+        sensor = sensor_list[0]
+        person = sensor.owner
+        imported_person = import_utils.import_person(person_data=person)
+        result = import_utils.import_sensor(sensor, imported_person)
+
+        assert type(result[0]) == models.Device2  # expet a device2 object to be returned
+        assert self.actual[0] == self.expected_1
+
+
+@pytest.mark.django_db
+class TestMigrateApiData:
```

Comment:
Ik zie nog her en der de term "Migrate" langs komen, misschien kun je ff zoeken om te zien of er nog andere plekken waar je beter "Convert" kunt gebruiken om het consistent te houden